site/content/docs/user/ingress.md

```diff
@@ -133,6 +134,40 @@ kubectl wait --namespace ingress-nginx \
 
 Refer [Using Ingress](#using-ingress) for a basic example usage.
 
+### Kong
+
+To run Kong ingress, create a ClusterRoleBinding granting full cluster
+admin privileges to the `default` ServiceAccount in `kube-system`:
```

Comment:
I'm not a Kong developer, so I don't have examples anywhere. :smile: I just tried to write up what I found in https://github.com/kubernetes-sigs/kind/issues/1294. I have no idea what it would take to get Kong running in Kind without this permissions change.

Reply:
do you mind if we don't move forward with this then?
mainly because this docs have been added from persons from the corresponding ingress controllers, so we can have some guarantee that the projects will take some responsibility ...

Reply:
I mean, I found #1294 super useful, and without it I wouldn't have gotten Kong working. It seems like an odd take to make your docs deliberately less useful and force people to spelunk in your PRs and issues, but I guess between #1294 and this PR it's more likely that those docs will wind up in a Google search, and the Kind docs will wind up accidentally being useful.

Feel free to close this if you really feel like an open source project rejecting documentation by the wrong author is the right course of action.

Reply:
> Feel free to close this if you really feel like an open source project rejecting documentation by the wrong author is the right course of action.

hey, don't take like that, that is not what I say.
If you still want, I'm happy to continue reviewing, but we need to polish some details, is not the first time that the persons that sends the PR gives up because it "just have to do more work", so it seems my comment was inappropriate or hurt your feelings, then I'm surry.

As you correctly say, documentation is important, so we want that all the docs are accurate and work, we shouldn't do the "admin privileges to the default service acconut", a quick look at the kong page have a link to a manifest that installs the rbacs and everything , similar to how other projects are doing in this page
https://docs.konghq.com/gateway-oss/2.4.x/kong-for-kubernetes/install/

```
YAML manifests
To deploy Kong via kubectl, use:

1kubectl apply -f https://bit.ly/kong-ingress-dbless
```

I'm not really familiar with Kong and I don't know what is the best method to install it, but I leave that to you to show other users the best way to do it